src/components/common/AddressInput/index.test.tsx

```diff
@@ -82,15 +82,6 @@ describe('AddressInput tests', () => {
   it('should validate the address on input', async () => {
     const { input, utils } = setup('')
 
-    act(() => {
-      fireEvent.change(input, { target: { value: `eth:${TEST_ADDRESS_A}` } })
-      jest.advanceTimersByTime(1000)
-    })
-
-    await waitFor(() =>
-      expect(utils.getByLabelText(`"eth" doesn't match the current chain`, { exact: false })).toBeDefined(),
-    )
```

Comment:
As I mentioned, the prefix and its validation shouldn’t be removed.

If the user enters “here’s my address: matic:0x1234…”, it should extract “matic:0x1234”. If it’s not a Polygon safe, an error should be shown.

Reply:
Added back the test case, refactored util function to support any prefix and show error.